cdist/conf/type/__systemd_unit/manifest

```diff
@@ -34,6 +34,11 @@ if [ -z "${source}" ] && [ "${state}" != "absent" ]; then
     exit 0
 fi
 
+# stdin is not propagated automatically to sub-objects
+if [ "x$source" = "x-" ]; then
```

Comment:
What is going on with this _x_ thing? Why is a simple `"${source}" = "-"` not enough?

Reply:
"old school", but I let somebody else elaborate on it :-)

Reply:
Uh, I see now (ref: https://stackoverflow.com/a/174288/1928832). But it'd be a bit redundant in this case, really.